snmp-exporter/snmp.yml

```diff
@@ -891,6 +891,18 @@ ddwrt:
           labelname: processID
           oid: 1.3.6.1.2.1.25.4.2.1.1
           type: gauge
+    - name: processRunArgs
+      oid: 1.3.6.1.2.1.25.4.2.1.5
+      type: DisplayString
+      indexes:
+        - labelname: processID
+          type: gauge
+      lookups:
+        - labels:
+            - processID
+          labelname: processID
+          oid: 1.3.6.1.2.1.25.4.2.1.1
+          type: gauge
```

Comment:
I think the problem with your `processRunArgs` definition was that it had an extra `.1` on the end.  Removing that let it work just fine.